packages/web/src/components/range/RangeInput.js

```diff
@@ -64,29 +58,34 @@ class RangeInput extends Component {
 			}
 		}
 
-		const currentValue = {
-			start: this.state.start,
-			end: this.state.end,
-			[name]: value,
-		};
-
-		const { value: valueProp, onChange } = this.props;
-		if (valueProp === undefined) {
+		if (this.isControlled) {
+			this.props.onChange({
+				...this.value,
+				[name]: value,
```

Comment:
We will also need to coerce the value in `handleInputChange` currently if we change the input start value we get a string in `value.start`.

Reply:
If we coerce the value before passing it to the `onChange`, won't we run into the same issue I described above, where we will always pass a coerced value to `input` elements (provided the user implementing `onChange` simply passes back to us what we gave them) , effectively making it impossible to `null` them out?

Reply:
I think the `min` and `max` makes a ton of sense